src/app/app.component.ts

```diff
@@ -7,4 +7,9 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'angular-training';
+  menuSelector = 'recipes';
```

Comment:
I would like to have enum here instead string literals. I will make your code more reliable/reusable etc